examples/with-react-native-web/apps/native/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "expo/tsconfig.base",
```

Comment:
**thought:** Should we enforce `@vercel/style-guide/typescript`?‏

Reply:
Yea good idea, I might do this in a followup and move all of the examples over to include our style guide.